easybuild/easyconfigs/s/StringTie/StringTie-2.1.7-GCC-10.3.0.eb

```diff
@@ -0,0 +1,38 @@
+# This file is an EasyBuild reciPY as per https://easybuilders.github.io/easybuild/
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'MakeCp'
+
+name = 'StringTie'
+version = '2.1.7'
+
+homepage = 'https://ccb.jhu.edu/software/stringtie/'
+description = 'StringTie is a fast and highly efficient assembler of RNA-Seq alignments into potential transcripts'
+
+toolchain = {'name': 'GCC', 'version': '10.3.0'}
+
+source_urls = ['https://github.com/gpertea/%(namelower)s/releases/download/v%(version)s']
```

Comment:
@SethosII isn't this the right version?: 
`https://ccb.jhu.edu/software/stringtie/dl/stringtie-2.1.7.tar.gz` ?

Reply:
@verdurin Ah ok, so they haven't updated it on the website (http://ccb.jhu.edu/software/stringtie/ has 2.1.6 as the latest and the changelog there is only up to 2.1.5). Both files are identical (I checked their SHA256). Should I change it to the url of the homepage?

Reply:
@SethosII I don't have strong feelings. If they make proper releases on Github, that's fine. 
Whichever is the most reliable.

Reply:
@verdurin Ok, then I would use GitHub and hope that they have a higher availability ;)